smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/serde/member/MemberShapeDecodeGenerator.kt

```diff
@@ -253,20 +253,21 @@ open class MemberShapeDecodeGenerator(
     }
 
     private fun resolveDocumentDefault(useZeroValue: Boolean, node: Node): String {
+        writer.addImport(SwiftDependency.SMITHY_JSON.target)
```

Comment:
SmithyJSON includes the literal initializers for Document.  Once you import it, you can initialize a Document with bare literals.